apparmor.d/groups/freedesktop/pulseaudio

```diff
@@ -113,10 +114,8 @@ profile pulseaudio @{exec_path} {
   /{usr/,}lib/@{multiarch}/pulse/gconf-helper mrix,
   /{usr/,}lib/pulse-*/modules/*.so mr,
 
-  /usr/share/applications/{,**} r,
   /usr/share/glib-2.0/schemas/gschemas.compiled r,
```

Comment:
Add to `abstractions/freedesktop.org.d`?

Reply:
Yes, I find that could be a good option.

Reply:
Wow, I'll move it in a separate PR:
```
grep -ri '/usr/share/glib-2.0/schemas/gschemas.compiled' apparmor.d | wc -l
131
```